pkgs/applications/audio/picard/default.nix

```diff
@@ -13,7 +22,17 @@ in pythonPackages.buildPythonApplication rec {
     sha256 = "0xalg4dvaqb396h4s6gzxnplgv1lcvsczmmrlhyrj0kfj10amhsj";
   };
 
-  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ];
+  nativeBuildInputs = [ gettext qt5.wrapQtAppsHook qt5.qtbase ]
```

Comment:
```suggestion
  nativeBuildInputs = [ gettext qt5.qtbase ]
```

Reply:
However, this wrap hook is still needed for the many qt plugins environments.